src/carplay.cs

```diff
@@ -807,13 +807,12 @@ interface CPWindow {
 		UILayoutGuide MapButtonSafeAreaLayoutGuide { get; }
 
 #if false
-// Apple is trying to use CPTemplateApplicationScene but it is not exposed. FB6231565
+// Apple is trying to use CPTemplateApplicationScene but it is not exposed. xamarin/maccore#1822
```

Comment:
Could you just use the full link? https://github.com/xamarin/maccore/issues/1822

Simpler